.kokoro/docker/docs/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ubuntu:20.10
+from ubuntu:20.04
```

Comment:
@busunkim96, @tmatsuo This is fighting with the change proposed by the Renovate bot, merged in PR #156.